### DISCUSSION TITLE
Laser grbl

### DISCUSSION THREAD

**Discussion by CacouRyo** [arkypita/LaserGRBL#2006] [Q&A]
Bonsoir
Lorsque dans grbl j importe un texte  d insckape en svg celui ne se place pas en bas a gauche mais au centre
Comment faire pour que le texte se trouve en bas a gauche
Merci de vos retour

**Comment by Tribo-2dot0** [comment]
selecione tudo, menu Editar > Ajustar Página à Seleção... salve.
